bin/start

```diff
@@ -34,7 +34,9 @@ EOF
 }
 
 runScriptWithSummon() {
-    summon --provider summon-conjur --environment $SUMMON_ENV -f ./summon/secrets.yml $1
+    # CONJUR_ACCOUNT is set in bootstrap.env to a test value, which isn't valid for
+    # conjurops.
+    CONJUR_ACCOUNT="conjur" summon --provider summon-conjur --environment $SUMMON_ENV -f ./summon/secrets.yml $1
```

Comment:
should these be two lines?
```
CONJUR_ACCOUNT="conjur"
summon --provider summon-conjur --environment $SUMMON_ENV -f ./summon/secrets.yml $1
```
?
Otherwise how does that summon line get executed if it is saved in a var?

Reply:
Bash allows setting an environment variable for just one command by prepending it to the line:
```
11:15 $ date
Tue 25 May 2021 11:15:24 BST
11:15 $ TZ="America/New_York" date
Tue 25 May 2021 06:15:27 EDT
```

Reply:
so it sets the command and runs it?

Reply:
Discussed in slack